weaver/core/drivers/corda-driver/README.md

```diff
@@ -1,9 +1,9 @@
-<!--
+driver-corda<!--
  Copyright IBM Corp. All Rights Reserved.
 
  SPDX-License-Identifier: CC-BY-4.0
  -->
-# corda-driver
+# driver-corda
```

Comment:
This is the title, so let's call this `Corda Driver` and not use the package name.